developer-docs/server-installation/core-services.md

```diff
@@ -76,7 +78,10 @@ Update **core_vault_sunbird_api_auth_token**, **core_vault_kong__test_jwt** and
 | KeycloakRealm | Creates Sunbird Realm. After the Sunbird realm is created, configure Keycloak by using the steps mentioned in the **Keycloak Configuration** section. |
 | Learner | Deploys the Learner Service. Handles user management and helps to search content. |
 | Lms | Deploys the Lms Service. It provides the APIs for lms functionality of Sunbird.|
-| Content | Deploys the content service. Helps to create content. |  
+| Content | Deploys the content service. Helps to create content. | 
+| certTemplate | Uploads static certification template to storage |
+| EncService | Deploys the encryption service. Helps to generate the encryption keys |
+| Cert | Deploys the cert service. Helps to issue the certificates |
```

Comment:
@Kaali09 should it be 'Deploys the certificate service' or is 'cert service' the name of the service?

Reply:
@Sunitashelar I have worked on your comment. please review.

Reply:
done